testCommon/src/test/resources/common.sh

```diff
@@ -36,16 +36,21 @@ function create_repo() {
   mkdir -p $dir
   cd $dir
   shift
-  git init $@
+  git init "$@"
   # TODO (#760): Enable running machete-status-branch hook in tests on Windows
   if [ "$(expr substr $(uname -s) 1 10)" != "MINGW64_NT" && "$(expr substr $(uname -s) 1 10)" != "MINGW32_NT" ]; then
     mkdir -p .git/hooks/
     local hook_path=.git/hooks/machete-status-branch
     echo "$status_branch_hook" > $hook_path
     chmod +x $hook_path
   fi
+  # `--local` (per-repository) is the default when writing git config... let's put it here for clarity anyway.
   git config --local user.email "circleci@example.com"
   git config --local user.name "CircleCI"
+  # Just to cover the scenario when on a developer machine,
```

Comment:
there is some some verb missing here, isn't? consider splitting this sentence into two (I don't fully get it) 😆

Reply:
Pls check now

Reply:
![bika-bika](https://user-images.githubusercontent.com/3383210/176424582-7bc4c488-ca21-4ab5-a82b-d46803648e4b.png)